composer.json

```diff
@@ -14,7 +14,7 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.3"
```

Comment:
set it also to include php 8

Reply:
i was going to do that separately, bumping the coding standard and PHPUnit too